docs/csharp/delegates-strongly-typed.md

```diff
@@ -67,14 +67,14 @@ The `out` modifier on the result generic type argument is covered in the
 article on covariance.
 
 There are variations of the `Func` delegate with up to
-[16 input arguments](../../core/api/system.func-17#System_Func_17).
+16 input arguments such as @System.Func%6017 .
```

Comment:
That's another funky format. Can we get that one changed to match our other generic format?